lib/pytis/dbdefs/db_pytis_menu.py

```
@@ -1341,6 +1341,7 @@ class PytisActionRightsFoldable(Base_PyFunction):
     @staticmethod
     def pytis_action_rights_foldable(shortname, column):
         shortname, column = args
+        import string
```

Ruff (0.13.2)

Redefinition of unused `string` from line 9

Remove definition: `string`

(F811)